src/CodeGen.js

```diff
@@ -98,17 +98,17 @@ void initLookups() {
   function emitInitConstantsCode() {
     mode = 'init-constants'
     return `
-${chunkedFunctions('initConstants', Model.constVars(),
-'  // Initialize constants.',
-'  initLookups();'
-)}
+${chunkedFunctions('initConstants', Model.constVars(), '  // Initialize constants.', '  initLookups();')}
```

Comment:
This would probably read better if we assigned the `chunkedFunctions` to a variable first and try to keep the args on multiple lines, but whatever.